src/java/fr/paris/lutece/portal/web/user/AdminUserJspBean.java

```diff
@@ -1003,6 +1003,10 @@ public String doModifyAdminUser( HttpServletRequest request ) throws AccessDenie
             user.setAccessibilityMode( strAccessibilityMode != null );
             user.setWorkgroupKey( strWorkgroupKey );
 
+            if ( isUserAuthorizedToChangeUserLevel( currentUser, userToModify ) ){
```

Comment:
ok, but I think it would be better to move this test back to the start of the method, where we control the rights: if the data is modified, and the current user does not have the rights, we should throw an AccessDeniedException (MESSAGE_NOT_AUTHORIZED).

and would it be possible to add a javadoc title block to the isUserAuthorizedToChangeUserLevel() method?